ccf_transaction_processor/README.md

```diff
@@ -0,0 +1,208 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+
+# Microsoft CCF based PDO Transaction Processor
+
+This folder contains software for PDO transaction processor (TP) based
+on Microsoft's CCF blockchain.  The software is located under
+$PDO_SOURCE_ROOT/transaction_processor/. The TP software is written and
+tested for CCF tag 0.9.2. Compatibility with other CCF versions is not
+guaranteed.
+
+The instructions below  can be used to build and deploy CCF based PDO
+TP. Make sure the following env variables are defined:
+
+1) PDO_SOURCE_ROOT : points to PDO local git repo directory.
+
+2) PDO_INSTALL_ROOT : CCF will be installed at PDO_INSTALL_ROOT/opt/pdo/ccf/.
+
+3) PDO_HOME : `export PDO_HOME=${PDO_INSTALL_ROOT}/opt/pdo/`.
+
+4) HOSTNAME : CCF's first node will be deployed at HOSTNAME:6006. One can simply set HOSTNAME to be the ip address
+of the VM that can be used to ping the VM from other machines.
+
+## Get CCF Source Code
+
+CCF tag 0.9.2 is included as a submodule within PDO. Download the
+submodule using the following commands:
+
+```bash
+cd $PDO_SOURCE_ROOT
+git submodule init
+git submodule update
+```
+
+## Install CCF Dependencies
+
+CCF/PDO combo has been tested under a scenario where CCF is deployed in
+a standalone VM, and where PDO cients/services are deployed at other VMs.
+Further, the CCF/PDO combo has been tested only for the CCF virtual enclave mode.
+The dependencies needed to deploy CCF in an Ubuntu 18.04 VM with virtual enclave mode can be installed by running
+the following command:
+
+```bash
+cd $PDO_SOURCE_ROOT/ccf_transaction_processor/CCF/getting_started/setup_vm/
+./setup_nodriver.sh
+```
+
+The above script works only when the VM is not behind any proxies. If
+there are proxies, make the following changes before executing the above
+command:
+
+A. In the setup_nosgx.sh file, modify the command
+`sudo add-apt-repository ppa:ansible/ansible -y`
+by adding the `-E` option. The modified command looks like
```

Comment:
shouldn't all of these be apt-add-repository rather than add-apt-repository?

Reply:
Hi Mic, Thank you for the question. No, it is add-apt-repository (http://manpages.ubuntu.com/manpages/trusty/man1/add-apt-repository.1.html). But I see another issue to fix, the file name is setup_nodriver and not setup_nosgx (that was for some older version of ccf, I thought i had fixed this, not sure)